### ISSUE TITLE
Installation script uses "macos" but release assets use "osx"

### ISSUE THREAD

**Issue by faunaee** [wjdp/htmltest#160]
# Describe the bug
The installation script served by https://htmltest.wjdp.uk determines the OS and ARCH to fetch, and for macOS 11 ("Bug Sur"), the determination is "macos". However, none of the release assets include that identifier. The correct determination would be "osx", based on current asset naming.

# To Reproduce
Steps to reproduce the behaviour:

```bash
$ curl https://htmltest.wjdp.uk | bash -s -- -d -b bin
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 10278  100 10278    0     0  68520      0 --:--:-- --:--:-- --:--:-- 68520
wjdp/htmltest info checking GitHub for latest tag
wjdp/htmltest debug http_download https://github.com/wjdp/htmltest/releases/latest
wjdp/htmltest info found version: 0.13.0 for v0.13.0/macos/amd64
wjdp/htmltest debug downloading files into /var/folders/lj/fgm_6fb14qg5qc7871zj_9f80000gn/T/tmp.UMkCy71U
wjdp/htmltest debug http_download https://github.com/wjdp/htmltest/releases/download/v0.13.0/htmltest_0.13.0_macos_amd64.tar.gz
wjdp/htmltest debug http_download_curl received HTTP status 404
```

# Expected behaviour
htmltest should be installed.

# Actual behaviour
htmltest is not installed

## Versions
 - OS: macOS 11 (Bug Sur)
 - htmltest: latest (0.13.0)

**Comment by wjdp** [comment]
Hi @faunaee Thanks for the report. This is because the script has been updated with the rename but the 0.14 release hasn't been released yet. I should be able to release 0.14 shortly which will fix this issue.

**Comment by wjdp** [comment]
@faunaee This should now be fixed with 0.14 release.

**Comment by faunaee** [comment]
I just tested, and 0.14 does indeed work correctly. Thanks for the fast turnaround!
